CSS/mainstyle.css

```diff
@@ -279,4 +319,202 @@ ul {
     top: 15px;
     right: 35px;
   }
+
+  .desktop-banner {
+    display: none;
+  }
+
+  .desktop-program-link {
+    display: none;
+  }
+
+  .partners {
+    display: none;
+  }
+
+  .desktop-footer {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) {
+  .mobile-nav {
+    display: none;
+  }
+
+  .desktop-nav-banner {
+    display: flex;
+    background: #3e3e3e;
+    color: white;
+    flex-direction: row;
+    justify-content: end;
+    align-items: center;
+    height: 30px;
+    font-size: 12px;
+    gap: 10px;
+    padding-right: 4em;
+  }
+
+  .desktop-nav {
+    background: white;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 0 10em 0 10em;
+    gap: 15%;
+  }
```

Comment:
- [ ] As I suggested previously [here](https://github.com/demesameneshoa/First-Capstone-Project/pull/4#discussion_r1204625699) kindly note that the horizontal scroll issue still persists so kindly make sure to fix it as it will help create a good user experience.